### ISSUE TITLE
License

### ISSUE THREAD

**Issue by boxerab** [myshell-ai/OpenVoice#152]
This is a fantastic project. I hate to be that guy, but any ballpark ETA on when the license will be changed to allow commercial use ?

**Comment by bitnom** [comment]
yeah. Even if you want to make money from the model, a llama-like license makes a lot more sense. It probably makes even more sense than charging for an API at this point.

**Comment by Zengyi-Qin** [comment]
In April or late March.

**Comment by Zengyi-Qin** [comment]
We will release a new version of OpenVoice that provides better audio quality and native multi-lingual support, and MIT licensed. We plan to release it in April or late March. The time is subject to change definitely within this Spring.

Also check out MeloTTS https://github.com/myshell-ai/MeloTTS . MIT licensed.

**Comment by boxerab** [comment]
Thanks, looking forward to the new version. I compared MeloTTS to OpenVoice on HuggingFace, and MeloTTS speech quality was better.
